ui/pages/settings/networks-tab/networks-list-item/networks-list-item.js

```diff
@@ -72,17 +73,23 @@ const NetworksListItem = ({
       }}
     >
       {isCurrentRpcTarget ? (
-        <IconCheck
-          className="networks-tab__content__icon-check"
-          color="var(--color-success-default)"
-          aria-label={t('active')}
-        />
+        <>
+          <Icon
+            name={ICON_NAMES.CHECK}
+            className="networks-tab__content__icon-check"
+            color={IconColor.successDefault}
+            aria-label={t('active')}
+          />
+        </>
       ) : (
-        <IconCheck
-          className="networks-tab__content__icon-check"
-          color="transparent"
-          aria-hidden="true"
-        />
+        <>
+          <Icon
+            name={ICON_NAMES.CHECK}
+            className="networks-tab__content__icon-check"
+            color={IconColor.successDefault}
```

Comment:
Need to update the color here.

Reply:
I did not understand? The colour is updated... What do I need to change?